### DISCUSSION TITLE
.Env file gets overwritten when application is in high demand (deployed on Windows

### DISCUSSION THREAD

**Discussion by ferventurart** [Blair2004/NexoPOS#269] [Gastro 4.x Ideas & Issues] [answered]
When the application is handling a lot of requests, overwrite the .env file broken the application. You could verify that by looking on storage > dotenv-editor->backups and you will see all the backups of .env and can view the sequence when is overwritten for any reason.

**Comment by magedfathi** [comment]
@ Ferventurart, Does it mean, system will stop while customers are using ?

**Comment by Blair2004** [comment, marked as answer]
NexoPOS 4.x keeps the .en file to sure everything is configured. Only a bad change on the .env file can make the application not able to startup. This is not related to the fact NexoPOS 4.x updates the .env file.

**Comment by Blair2004** [comment]
After further verification, i've noticed that the error is proven. I'll convert this into an issue. Thank you.
